app/views/users/show.html.erb

```diff
@@ -1,8 +1,8 @@
-<%= link_to "Create a new post", user_post_new_path(@user.id), class: 'see-posts-button' %>
+<%= link_to "Create a new post", new_post_path(@user.id), class: 'see-posts-button' %>
```

Comment:
- [ ] Unfortunately, I cannot create a new user. After I signup and try to log in I get the `You have to confirm your email address before continuing.` But I don't receive any confirmation email. Please let me know if I am missing something

Reply:
@GzimAsani thank you for the review, please check your terminal for the link to confirm your email

Reply:
There are no issues with the email confirmation that you requested changes for, everything works fine